app/assets/javascripts/map.js

```diff
@@ -1,28 +1,100 @@
+/* global google $ */
+
 const info =
 `<div id = content>
-  <div id = "photo" style="float: left">
-    <img src="https://www.dogstrust.org.uk/sponsor/_dogs/digby-assets/v800_digby01.jpg" alt="dog" height="160" width="120">
+  <div id = 'photo' style='float: left'>
+    <img src='https://www.dogstrust.org.uk/sponsor/_dogs/digby-assets/v800_digby01.jpg' alt='dog' height='160' width='120'>
```

Comment:
It may be a good idea to have this image as part of our application not to be dependant on an external application